lib/pure/collections/intsets.nim

```diff
@@ -7,698 +7,18 @@
 #    distribution, for details about the copyright.
 #
 
-## The ``intsets`` module implements an efficient `int` set implemented as a
-## `sparse bit set`:idx:.
-##
-## **Note**: Currently the assignment operator ``=`` for ``IntSet``
-## performs some rather meaningless shallow copy. Since Nim currently does
-## not allow the assignment operator to be overloaded, use `assign proc
-## <#assign,IntSet,IntSet>`_ to get a deep copy.
-##
+## Deprecated by the generic `OrdSet` for ordinal sparse sets.
 ## **See also:**
-## * `sets module <sets.html>`_ for more general hash sets
+## * `ordinal sets module <ordsets.html>`_ for more general int sets
 
 import std/private/since
-import hashes
+import std/ordsets
+export ordsets
 
 type
-  BitScalar = uint
+  IntSet* = OrdSet[int]
 
-const
-  InitIntSetSize = 8              # must be a power of two!
-  TrunkShift = 9
-  BitsPerTrunk = 1 shl TrunkShift # needs to be a power of 2 and
-                                  # divisible by 64
-  TrunkMask = BitsPerTrunk - 1
-  IntsPerTrunk = BitsPerTrunk div (sizeof(BitScalar) * 8)
-  IntShift = 5 + ord(sizeof(BitScalar) == 8) # 5 or 6, depending on int width
-  IntMask = 1 shl IntShift - 1
+proc toIntSet*(x: openArray[int]): IntSet {.since: (1, 3), inline.} = toOrdSet[int](x)
```

Comment:
`(1,5,1)` now

Reply:
toIntset was public and had the since 1.3 from before, so shouldn't we keep that version info?

Reply:
misread